src/resource.js

```diff
@@ -25,7 +25,8 @@ export default class Resource {
 		this.collectionRoot = collectionRoot;
 		this.itemRoot = itemRoot;
 
-		this.descriptor = Container.get(this.service);
+		const obj = {service: 'json-placeholder', prefix: 'http://jsonplaceholder.typicode.com', agent: 'restinga-node/1.0.0 (https://github.com/jay-ess/restinga-node)'};
```

Comment:
Name it `const descriptior` to be more clear.

Don't make it a one-liner as it's too big.

Reply:
done